using/rendering-questions.md

```diff
@@ -2,59 +2,238 @@
 
 The PIE [Packaging Tool](packaging-questions.md) is used to assemble and package the Javascript and HTML into files needed to render an Assessment Item in the browser.
 
-An Assessment Item packed by the Packaging Tool may have some or all of the following files:
+The command uses 2 input files: 
 
-| File              | Description                                                        |
+| File              | Description                                                        | 
 |-------------------|--------------------------------------------------------------------|
-| pie.js            | Single file containing all code and config to render the item      |
-| config.json       | JSON data that defines the questions & interaction PIEs in an item |
-| index.html        | Markup for adding the PIEs to html document                        |
-| pie-view.js       | Assembled Javascript for rendering the UI for the Assessment Item  |
-| pie-controller.js | Packaged controller code for the PIEs defined in the config        |
+| config.json       | JSON data that defines the questions & interaction PIEs in an item | 
+| index.html        | Markup defining the layout of the pies inside a pie-player         |
 
 
-> By convention, these files will be contained in a directory named with a unique id for the assessment item.
+It generates the following files: 
 
-## Simple Usage - Client Side Only
+| File              | Description                                                        | usecase   |
+|-------------------|--------------------------------------------------------------------|-----------|
+| pie.js            | Single file containing everything needed to render the item        | simple    |
+| pie-view.js       | Assembled Javascript for rendering the UI for the Assessment Item  | advanced  |
+| pie-controller.js | Packaged controller code for the PIEs defined in the config        | advanced  |
 
-The simplest way to load a PIE-formatted item is to use the `pie.js` file. This file includes everything needed to render the item in a user's browser. 
+We have 2 usecases above. `simple` is the most convenient but lacks reusability, `advanced` requires more effort to use but allows for reusability. See below for more information on reuse.
```

Comment:
would not add this, again this page is for ppl rendering. reusability may mean nothing to them if all they want is packaged item.